### ISSUE TITLE
Manifest Tasks does not support the manifestOutputFile property any more, please use the manifestOutputDirectory instead.

### ISSUE THREAD

**Issue by gchristov** [splatte/gradle-android-appiconoverlay#8]
Getting this error on Gradle 3 with Android Studio 3 and had to disable this plugin. Any ideas why this is happening?

Here's the suggested link for more information:

```
For more information, please check https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html
```

**Comment by splatte** [comment]
The property `manifestOutputFile` was removed, which the plugin uses here https://github.com/splatte/gradle-android-appiconoverlay/blob/eacffa533b309b89756cd2b119e5942d356ccea9/src/main/groovy/com/github/splatte/android/AppIconOverlayPlugin.groovy#L26 to locate the manifest file. To be compatible with android gradle plugin 3, `manifestOutputFile()` must now be used instead to locate the file (https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#manifestOutputFile).

**Comment by gchristov** [comment]
Thanks! Do you think we can have an update with this change?

**Comment by splatte** [comment]
Not sure I will find the time myself, but I am more than happy to try out and include a PR to make a new version

**Comment by MinaSamirCap** [comment]
I have the same issue and can not solve it .. :( :(

**Comment by splatte** [comment]
thanks for your PR, @rocket0423

I have released version 2.0 of the plugin. try it out and let me know if there are any problems.

**Comment by gchristov** [comment]
Just tried this and the error is indeed resolved! However, now I have a different issue which is related to the font size of the overlay where the text gets cropped. I'll open a new issue for this.
